ci/azure-cross-compile.yml

```diff
@@ -0,0 +1,52 @@
+parameters:
+  cmd: test
+  rust_version: stable
+
+jobs:
+- job: ${{ parameters.name }}
+  displayName: Cross
+  strategy:
+    matrix:
+      i686:
+        vmImage: ubuntu-16.04
+        target: i686-unknown-linux-gnu
+      armv7:
+        vmImage: ubuntu-16.04
+        target: armv7-unknown-linux-gnueabihf
+      powerpc:
+        vmImage: ubuntu-16.04
+        target: powerpc-unknown-linux-gnu
```

Comment:
I removed `RUST_TEST_THREADS=1` because it looks like cross is already setting it.
* https://github.com/rust-embedded/cross/blob/540dc4f5a1ca8d6d95bbf4da66ce763a2352d36f/docker/powerpc-unknown-linux-gnu/Dockerfile#L53
* https://github.com/rust-embedded/cross/blob/540dc4f5a1ca8d6d95bbf4da66ce763a2352d36f/docker/powerpc64-unknown-linux-gnu/Dockerfile#L49